content/numpy/concepts/math-methods/terms/cos/cos.md

```diff
@@ -0,0 +1,75 @@
+---
+Title: 'NumPy cos'
+Description: 'Returns the cos of the argument.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'NumPy'
+  - 'Trig'
+  - 'Math'
+  - 'Methods'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
```

Comment:
```suggestion
---
Title: '.cos()'
Description: 'Computes the cosine of each element in an array or a single value.'
Subjects:
  - 'Computer Science'
  - 'Data Science'
Tags:
  - 'Arrays'
  - 'NumPy'
CatalogContent:
  - 'learn-python-3'
  - 'paths/computer-science'
---
```

Reply:
The description could be modified as given above, and just add two relatable tags for any PR